### PR TITLE
fix(docs): Fix two typos in design.md

### PR DESCRIPTION
**Pull request by haslersn**
* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

- "scaled up" -> "scaled out"
- "as approximately as" -> "approximately as"

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ The compactor also does additional batch processing such as down-sampling and ap
 
 ## Scaling
 
-None of the Thanos components provides any means of sharding. The only explicitly scalable component are query nodes, which are stateless and can be scaled up arbitrarily. Scaling of storage capacity is ensured by relying on an external object storage system.
+None of the Thanos components provides any means of sharding. The only explicitly scalable component are query nodes, which are stateless and can be scaled out arbitrarily. Scaling of storage capacity is ensured by relying on an external object storage system.
```
Comment (bwplotka) [inline docs/design.md:141]: > None of the Thanos components provides any means of sharding.

Wow, this does not hold truth anymore ;p 

I think at this point we need to rename it as `Initial Design`? (:
cc @brancz

**Review by bwplotka — APPROVED** [review]
LGTM (: Can you fix DCO?

**Comment by haslersn** [comment]
> Can you fix DCO?

What's DCO?

**Comment by soniasingla** [comment]
@haslersn check your tests are failing, visit [here](https://github.com/thanos-io/thanos/pull/3445/checks?check_run_id=1400722533) The instructions are written how to sign it correctly!

And to read about DCO, check [this](https://probot.github.io/apps/dco/) out, hope it helps you :)

**Review by bwplotka — APPROVED** [review]
Thanks! 

And thanks @soniasingla for helping out!

**Pull request merged**